compiler-rt/lib/interception/tests/interception_win_test.cpp

```diff
@@ -17,6 +17,7 @@
 // Too slow for debug build
 #if !SANITIZER_DEBUG
 #if SANITIZER_WINDOWS
+#if !SANITIZER_WINDOWSARM64
```

Comment:
Similarly a comment for the future readers of this code. Also consider change SANITIZER_WINDOWSARM64 to SANITIZER_WINDOWS_ARM64 or something shorter and readable.

Reply:
Added a comment. Changed the name to `SANITIZER_WINDOWS_ARM64`. The reason to disable is because all these tests assume x86/x86_64 assembly.  Keeping these on is just going to make the llvm checks run on tests that will always fail.